docs/_docs/reference/changed-features/structural-types.md

```diff
@@ -66,38 +84,38 @@ The `person.name` and `person.age` selections are translated by
 the Scala compiler to:
 
 ```scala
-  person.selectDynamic("name").asInstanceOf[String]
-  person.selectDynamic("age").asInstanceOf[Int]
+person.selectDynamic("name").asInstanceOf[String]
+person.selectDynamic("age").asInstanceOf[Int]
 ```
 
 Besides `selectDynamic`, a `Selectable` class sometimes also defines a method `applyDynamic`. This can then be used to translate function calls of structural members. So, if `a` is an instance of `Selectable`, a structural call like `a.f(b, c)` would translate to
 
 ```scala
-  a.applyDynamic("f")(b, c)
+a.applyDynamic("f")(b, c)
 ```
 
 ## Using Java Reflection
 
-Structural types can also be accessed using [Java reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html). Example:
-
+If a structural type's base type does not have Selectable as a supertype, as below, it uses [Java reflection](https://www.oracle.com/technical-resources/articles/java/javareflection.html) instead:
+Write about {} meaning with an Any at the beginning
```

Comment:
?

Reply:
Sorry that was a TODO that slipped through
```suggestion

```